docs/pages/admin-guides/management/diagnostics/metrics.mdx

```diff
@@ -14,12 +23,168 @@ collect from your Teleport cluster.
 This will enable the `http://127.0.0.1:3000/metrics` endpoint, which serves the
 metrics that Teleport tracks. It is compatible with [Prometheus](https://prometheus.io/) collectors.
 
-The following metrics are available:
+## Backend operations
+
+A Teleport cluster cannot function if the Auth Service does not have a healthy
+cluster state backend. You need to track the ability of the Auth Service to read
+from and write to its backend. 
+
+The Auth Service can connect to [several possible
+backends](../../../reference/backends.mdx). In addition to Teleport backend
+metrics, you should set up monitoring for your backend of choice so that, if
+these metrics show problematic values, you can correlate them with metrics on
+your backend infrastructure.
+
+### Backend operation throughput and availability
+
+On each backend operation, the Auth Service increments a metric. Backend
+operation metrics have the following format:
+
+```text
+teleport_backend_<METRIC_NAME>[_failed]_total
+```
+
+If an operation results in an error, the Auth Service adds the `_failed` segment
+to the metric name. For example, successfully creating a record increments the
+`teleport_backend_write_requests_total` metric. If the create operation fails,
+the Auth Service increments `teleport_backend_write_requests_failed_total`
+instead.
+
+The following backend operation metrics are available:
+
+|Operation|Incremented metric name|
+|---|---|
+|Create an item|`write_requests`|
+|Modify an item, creating it if it does not exist|`write_requests`|
+|Update an item|`write_requests`|
+|Conditionally update an item if versions match|`write_requests`|
+|List a range of items|`batch_read_requests`|
+|Get a single item|`read_requests`|
+|Compare and swap items|`write_requests`|
+|Delete an item|`write_requests`|
+|Conditionally delete an item if versions match|`write_requests`|
+|Write a batch of updates atomically, failing the write if any update fails|Both `write_requests` and `atomic_write_requests`|
+|Delete a range of items|`batch_write_requests`|
+|Update the keepalive status of an item|`write_requests`|
+
+You can use these metrics to define an availability formula, i.e., the
+percentage of reads or writes that succeeded. Take the sum of requests that
+succeeded (including batch requests) over the total sum of requests, multiplied
+by 100. If your backend begins to appear unavailable, you can investigate your
+backend infrastructure.
```

Comment:
While this does give you calculation of availability, be warned that without calculating for `backend_write_requests_failed_precondition_total` it may be confusing as to why availability is not 100%.  This could be confusing to users.

Sometimes pictures are helpful here's the difference.  I abbreviated the prom query:
`1- (sum(rate(backend_write_requests_failed_total / sum(rate(backend_write_requests_total))`
![Screenshot 2024-09-30 at 3 34 28 PM](https://github.com/user-attachments/assets/86ca56d4-b41f-4856-acca-e5fcd21bff74)
`1- (sum(rate(backend_write_requests_failed_total -sum(rate(teleport_backend_write_requests_failed_precondition_total)) / sum(rate(backend_write_requests_total))`
![Screenshot 2024-09-30 at 3 34 42 PM](https://github.com/user-attachments/assets/8976a2f0-06b6-4063-93eb-e7f687068e1a)